src/pal/inc/pal.h

```diff
@@ -1637,13 +1637,6 @@ PALAPI
 PAL_LocalHandleToRemote(
```

Comment:
Can you please remove this prototype as well since you have deleted the function implementation?